systemtest/src/test/java/io/strimzi/systemtest/oauth/OauthPlainST.java

```diff
@@ -0,0 +1,293 @@
+/*
+ * Copyright 2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.oauth;
+
+import io.fabric8.openshift.client.OpenShiftClient;
+import io.strimzi.api.kafka.model.CertSecretSourceBuilder;
+import io.strimzi.api.kafka.model.KafkaResources;
+import io.strimzi.systemtest.Constants;
+import io.strimzi.systemtest.utils.StUtils;
+import io.strimzi.test.TestUtils;
+import io.strimzi.test.executor.Exec;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.client.WebClient;
+import io.vertx.ext.web.client.WebClientOptions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Random;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import static io.strimzi.systemtest.Constants.HTTP_BRIDGE_DEFAULT_PORT;
+import static io.strimzi.systemtest.Resources.deployLoadBalancerService;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.greaterThan;
+
+public class OauthPlainST extends OauthST {
+
+    @Test
+    void testProducerConsumer() {
+        testMethodResources().producerWithOauth(oauthTokenEndpointUri, TOPIC_NAME, KafkaResources.plainBootstrapAddress(CLUSTER_NAME)).done();
+        testMethodResources().consumerWithOauth(null, oauthTokenEndpointUri, TOPIC_NAME, KafkaResources.plainBootstrapAddress(CLUSTER_NAME)).done();
+
+        String producerPodName = kubeClient().listPodsByPrefixInName("hello-world-producer-").get(0).getMetadata().getName();
+        String producerMessage = "Sending messages \"Hello world - 9\"";
+
+        StUtils.waitUntilMessageIsInPodLogs(producerPodName, producerMessage);
+
+        String producerLogs = kubeClient().logs(producerPodName);
+        int messageCount = 10;
+
+        for (int i = 0; i < messageCount; i++) {
+            assertThat("Producer doesn't send message" + i, producerLogs, containsString("Sending messages \"Hello world - " + i + "\""));
+        }
+
+        String consumerPodName = kubeClient().listPodsByPrefixInName("hello-world-consumer-").get(0).getMetadata().getName();
+        String consumerMessage = "value: \"Hello world - 9\"";
+
+        StUtils.waitUntilMessageIsInPodLogs(consumerPodName, consumerMessage);
+
+        String consumerLogs = kubeClient().logs(consumerPodName);
+
+        for (int i = 0; i < messageCount; i++) {
+            assertThat("Producer doesn't send message" + i, consumerLogs, containsString("value: \"Hello world - " + i + "\""));
+        }
+    }
+
+    @Test
+    void testProducerConsumerStreams() {
```

Comment:
You have already the test, so it is maybe not worth deleting it now. But I'm not sure we need to test the Streams like this.